tests/compute/global-type-param.slang

```diff
@@ -1,6 +1,6 @@
 //TEST(smoke,compute):COMPARE_COMPUTE:-xslang -use-ir
 //TEST_INPUT:ubuffer(data=[0], stride=4):dxbinding(0),glbinding(0),out
-//TEST_INPUT:type Impl
+//TEST_INPUT:type Wrapper<Impl>
```

Comment:
We should be a bit careful about changing existing tests, and should instead prefer to create new ones that test the expanded functionality. Otherwise we risk removing coverage we used to have.